account_invoice_auto_send_mail/models/account_invoice.py

```diff
@@ -0,0 +1,88 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+import logging
+
+from odoo import api, fields, models
+from datetime import datetime
+_logger = logging.getLogger(__name__)
+
+
+class AccountInvoice(models.Model):
+    _inherit = 'account.invoice'
+
+    date_invoice_send_mail = fields.Datetime(
+        string='Date invoice send mail'
+    )
+
+    @api.one
+    def account_invoice_auto_send_mail_item_real(self, mail_template_id, author_id):
+        _logger.info('Operations account_invoice_auto_send_mail_item_real invoice ' + str(self.id))
```

Comment:
Please use only proper string injection where possible., I.e., '%s/%s' % (x, y), and not x + '/' + y